libs/accounts.js

```diff
@@ -13,61 +13,71 @@ module.exports = class Accounts {
     
     accountInfo(address){
         return accounts.getAccount(address)
-            .then(data => {
-                return data.body.result;
-            })
-            .catch(err => {
-                throw(err)
-            })
+        .then(data => {
+            return data.body.result;
+        })
+        .catch(err => {
+            throw(err)
+        })
+    }
```

Comment:
I think its better that you return this part to original one to see minimum changes in your commit!

Reply:
Yeah you are right ! But the code was not aligned. This commit makes it more aligned so that in future there will be minimum changes. Thanks

Reply:
thanks!